### PR TITLE
#170947645 Config Sequelize and Postgres

### PR DESCRIPTION
**Pull request by Kagorora**
#### What does this PR do?

- This PR configures Postgres and setup Sequelize to create tables

#### Description of Task to be completed?

- I should be able to create all needed tables in the database by running a single script using sequelize and populate those tables with dummy datas

#### How should this be manually tested?

- run `git clone https://github.com/andela/knights-bn-backend.git` to clone the project

- run `git checkout ch-Setup-Sequelize-170947645` to go the appropriate branch

- run `npm i` to install all necessary packages

- create a database  locally 

- complete your `.env` with the name of your database `DB_NAME=name of your database`

- run `npm run createTables` to create automatically all tables

- run `npm run dropTables` to drop all the tables

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [https://www.pivotaltracker.com/story/show/170947645](https://www.pivotaltracker.com/story/show/170947645)

- [https://www.pivotaltracker.com/story/show/170947644](https://www.pivotaltracker.com/story/show/170947644)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

**Review by Niyonsengaeric — APPROVED** [review]

**Review by Moise1 — APPROVED** [review]

**Review by EugeneMunya — APPROVED** [review]
Looks good

**Review by T2Wil — APPROVED** [review]

File: src/db/migrations/20200205203623-create-client.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Clients', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
```
Comment (MuhanguziDavid) [inline src/db/migrations/20200205203623-create-client.js]: add a trailing comma here

File: src/db/migrations/20200205203623-create-client.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    name: {
+      type: Sequelize.STRING
```
Comment (MuhanguziDavid) [inline src/db/migrations/20200205203623-create-client.js]: trailing comma here as well and on other instances like this

Comment (Kagorora) [reply src/db/migrations/20200205203623-create-client.js]: I have added them. Please check

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**